sdk/communication/AzureCommunicationChat/AzureCommunicationSignaling.framework/Headers/AzureCommunicationSignaling-Swift.h

```diff
@@ -0,0 +1,222 @@
+// Generated by Apple Swift version 5.3 (swiftlang-1200.0.29.2 clang-1200.0.30.1)
+#ifndef AZURECOMMUNICATIONSIGNALING_SWIFT_H
```

Comment:
The framework file itself should not be committed to the repo.

Reply:
removed, will work on publishing the trouter signaling library through github release instead